examples/third_party_integration/helm/helm_test.go

```diff
@@ -56,7 +56,7 @@ func TestHelmChartRepoWorkflow(t *testing.T) {
 		Assess("Deployment is running successfully", func(ctx context.Context, t *testing.T, config *envconf.Config) context.Context {
 			deployment := &appsv1.Deployment{
 				ObjectMeta: v1.ObjectMeta{
-					Name:      "nginx-nginx-ingress",
+					Name:      "nginx-nginx-ingress-controller",
```

Comment:
@vladimirvivien  should we have a dedicated chart pushed to the helm registry that is just meant for our test and use that instead in order to avoid failures like this one for example in the future ?

cc @cpanato

Reply:
sounds good, i can do that if you all want